packages/js-polyfills/BUCK

```diff
@@ -21,8 +21,8 @@ yarn_workspace(
 )
 
 rn_library(
-    name = "polyfills",
```

Comment:
Commenting here but valid for the whole PR:

Let's undo the BUCK update and the path change. It will make easier for us to import and merge this change 👍 

I've also posted an update on the monorepo effort there:
- https://github.com/facebook/react-native/issues/34692